### PR TITLE
Improve README with table of results and countries

### PR DESCRIPTION
**Pull request by davebulaval**
#70

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,40 +8,59 @@
...
-}
-```
-
-
-------------------
-
-## Getting started: 
+Deepparse is compatible with the __latest version of PyTorch__ and  __Python >= 3.6__.
+
+### Country and Results
```
Comment (MAYAS3) [inline README.md:19]:
```suggestion
### Countries and Results
```

File: README.md [hunk 2]
```diff
@@ -8,40 +8,59 @@
...
+
+The following table presents the accuracy of the 20 countries we used during training for both our models.
```
Comment (MAYAS3) [inline README.md:21]:
```suggestion
The following table presents the accuracy on the 20 countries we used during training for both our models.
```

File: docs/source/index.rst [hunk 3]
```diff
@@ -24,29 +24,220 @@ Read the documentation at `deepparse.org <https://deepparse.org>`_.
...
-   @misc{yassine2020leveraging,
-       title={{Leveraging Subword Embeddings for Multinational Address Parsing}},
-       author={Marouane Yassine and David Beauchemin and François Laviolette and Luc Lamontagne},
-       year={2020},
-       eprint={2006.16152},
-       archivePrefix={arXiv}
-   }
-
-and this one for the package;
+Country and Results
```
Comment (MAYAS3) [inline docs/source/index.rst:27]:
```suggestion
Countries and Results
```

File: docs/source/index.rst [hunk 4]
```diff
@@ -24,29 +24,220 @@ Read the documentation at `deepparse.org <https://deepparse.org>`_.
...
+===================
+
+The following table presents the accuracy of the 20 countries we used during training for both our models.
```
Comment (MAYAS3) [inline docs/source/index.rst:30]:
```suggestion
The following table presents the accuracy on the 20 countries we used during training for both our models.
```

File: docs/source/index.rst [hunk 5]
```diff
@@ -24,29 +24,220 @@ Read the documentation at `deepparse.org <https://deepparse.org>`_.
...
+			- 99.84
+		*	- Spain
+			- 99.70
+			- 99.79
+			- Czechia
+			- 99.46
+			- 99.83
+
+
+We have also made a zero-shot evaluation of our models using 41 other countries; the results are shown in the next table.
```
Comment (MAYAS3) [inline docs/source/index.rst:103]:
```suggestion
We have also made a zero-shot evaluation of our models using data from 41 other countries; the results are shown in the next table.
```

File: README.md [hunk 6]
```diff
@@ -8,40 +8,59 @@
...
+| Mexico      |          99.54 |       99.60 | Australia      |          99.62 |       99.74 |
+| Finland     |          99.75 |       99.87 | Netherlands    |          99.50 |       99.84 |
+| France      |          99.54 |       99.50 | United Kingdom |          99.54 |       99.62 |
+| Russia      |          98.71 |       99.49 | Norway         |          99.40 |       98.71 |
+| Switzerland |          99.48 |       99.61 | Poland         |          99.64 |       99.83 |
+| Brazil      |          99.33 |       99.24 | Denmark        |          99.65 |       99.84 |
+| Spain       |          99.70 |       99.79 | Czechia        |          99.46 |       99.83 |
+
+
+We have also made a zero-shot evaluation of our models using 41 other countries; the results are shown in the next table.
```
Comment (MAYAS3) [inline README.md:37]:
```suggestion
We have also made a zero-shot evaluation of our models using data from 41 other countries; the results are shown in the next table.
```

File: models_evaluation/tools.py [hunk 7]
```diff
@@ -0,0 +1,147 @@
...
+
+    for _, row in table.iterrows():
+        for idx, data in enumerate(list(row)):
+            if idx == 0:
+                string = string + new_line_prefix + "*" + f"\t- {data}\n"
+            else:
+                string = string + new_line_prefix + f"\t- {data}\n"
+
+    with open(os.path.join(table_dir, f"{data_type}_table.rst"), "w", encoding="utf-8") as file:
+        file.writelines(string)
```
Comment (MAYAS3) [inline models_evaluation/tools.py:147]: Nice one!

Comment (davebulaval) [reply models_evaluation/tools.py:147]: Yeah, no way I would copy pasta all the lines one by one.

**Pull request merged**